modules/imgcodecs/src/grfmt_tiff.hpp

```diff
@@ -77,7 +77,10 @@ enum  TiffTag
     TIFF_TAG_SAMPLES_PER_PIXEL = 277,
     TIFF_TAG_ROWS_PER_STRIP = 278,
     TIFF_TAG_PLANAR_CONFIG = 284,
-    TIFF_TAG_COLOR_MAP = 320
+    TIFF_TAG_COLOR_MAP = 320,
+    TIFF_TAG_RESOLUTION_UNIT = 296,
+    TIFF_TAG_X_RESOLUTION = 282,
+    TIFF_TAG_Y_RESOLUTION = 283
```

Comment:
These enums must go away completely (new and buggy old).
All these values are defined in "tiff.h". This header is private, so there is no sense to duplicate these unused values here.